components/kyma-environment-broker/internal/storage/driver/postsql/operation.go

```diff
@@ -450,6 +513,36 @@ func (s *operations) ListUpgradeKymaOperationsByOrchestrationID(orchestrationID
 	return ret, count, totalCount, nil
 }
 
+func (s *operations) ListOperationsByOrchestrationID(orchestrationID string, filter dbmodel.OperationFilter) ([]internal.Operation, int, int, error) {
+	session := s.NewReadSession()
+	var (
+		operations        = make([]dbmodel.OperationDTO, 0)
+		lastErr           error
+		count, totalCount int
+	)
+	err := wait.PollImmediate(defaultRetryInterval, defaultRetryTimeout, func() (bool, error) {
```

Comment:
I know that existing methods are doing the polling, but maybe we can consider to not do that in that (Storage) layer?